### PR TITLE
added Boilerplate for  FastAPI

### PR DESCRIPTION
**Pull request by marlin-dev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -191,6 +191,8 @@ Compute:
 - [fastapi-gino-arq-uvicorn](https://github.com/leosussan/fastapi-gino-arq-uvicorn) - Tempate for a high-performance async REST API, in Python. FastAPI + GINO + Arq + Uvicorn (w/ Redis and PostgreSQL).
 - [FastAPI and React Template](https://github.com/Buuntu/fastapi-react) - Full stack cookiecutter boilerplate using FastAPI, Typescript, Docker, PostgreSQL, and React.
 - [FastAPI Nano](https://github.com/rednafi/fastapi-nano) - Simple FastAPI template with factory pattern architecture.
+- [FastAPI-Gino-Alembic](https://github.com/Turall/FastApi-boilerplate) - Boilerplate for RestApi on FastApi with gino and alembic configuration.
+
```
Comment (Kludex) [inline README.md:195]:
```suggestion
```

**Comment by Kludex** [comment]
Even with the name "boilerplate" on the title, it doesn't seem a boilerplate to me...

**Comment by marlin-dev** [comment]
> Even with the name "boilerplate" on the title, it doesn't seem a boilerplate to me...

why it does not seem to you boilerplate ? This boilerplate has been designed for a microservices architecture. We have some applications running with this boilerplate.

**Comment by marlin-dev** [comment]
@mjhea0 this PR still open, is it going to be accepted?

**Comment by mjhea0** [comment]
@marlin-dev I like the project structure; however, I think this would be much more beneficial to users if it was wrapped with a generator like https://github.com/cookiecutter/cookiecutter so a user could customize it. At the very least, can you beef up the docs and add the basic commands someone would need in order to create migrations and move the schema up or down, etc. There's also some buried environment variables that took a bit for me to uncover. Please add that to the README as well or use a Docker Compose file so it's easier for someone to know _how_ to override the defaults.

**Comment by marlin-dev** [comment]
> @marlin-dev I like the project structure; however, I think this would be much more beneficial to users if it was wrapped with a generator like https://github.com/cookiecutter/cookiecutter so a user could customize it. At the very least, can you beef up the docs and add the basic commands someone would need in order to create migrations and move the schema up or down, etc. There's also some buried environment variables that took a bit for me to uncover. Please add that to the README as well or use a Docker Compose file so it's easier for someone to know _how_ to override the defaults.

Thank you for your response, we will take this into account and add these additional features soon.

**Pull request closed without merging**